frontends/parsers/p4/p4parser.ypp

```diff
@@ -459,10 +459,14 @@ annotation
           $$ = new IR::Annotation(@1, *$2, body); }
     | "@" name "(" annotationBody ")"
         { $$ = new IR::Annotation(@1, *$2, *$4); }
```

Comment:
Do you need to add `, false` to args?

Reply:
If your body is a sequence of tokens then by definition it's not structured. This constructor will set `structured` to false.

Reply:
OK,  thanks.